_posts/0000-01-02-krisha-patel.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Learning GitHub
So far I have learned:

Basics of Github
Assigned myself
created a branch
created a pull request
And now, I am trying to edit my pull request to fix any errors.

Thank you everyone